packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/notification/NotificationController.java

```diff
@@ -0,0 +1,65 @@
+package software.uncharted.terarium.hmiserver.controller.notification;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import software.uncharted.terarium.hmiserver.models.notification.NotificationGroup;
+import software.uncharted.terarium.hmiserver.security.Roles;
+import software.uncharted.terarium.hmiserver.service.CurrentUserService;
+import software.uncharted.terarium.hmiserver.service.notification.NotificationService;
+
+@RequestMapping("/notification")
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class NotificationController {
```

Comment:
Should there be a `deleteNotificationGroup` or `acknowledgeNotificationGroup` function to clear groups that are complete and "acknowledged" by the user.

And maybe a cron job that deletes a group that was completed and is something like a month old (though probably on the NotificationService and not this Controller)

Thoughts for "completeness"

Reply:
Added this in.